framework/infrastructure/gcp/compute.py

```diff
@@ -558,6 +568,78 @@ def get_backend_service_backend_health(self, backend_service, backend):
             .execute()
         )
 
+    def create_serverless_neg(self, name: str, region: str, service_name: str):
+        """Creates a serverless NEG.
+
+        Args:
+            name: The name of the NEG.
+            region: The region in which to create the NEG.
+            service_name: The name of the Cloud Run service.
+            service_name format is "namespaces/{namespace}/services/{service}"
+
+        Returns:
+            The NEG selfLink URL
+        """
+        neg_body = {
+            "name": name,
+            "networkEndpointType": "SERVERLESS",
+            "cloudRun": {"service": service_name},
+        }
+
+        try:
+            logger.info("Creating serverless NEG %s in %s", name, region)
+            self.api.regionNetworkEndpointGroups().insert(
```

Comment:
This should use `self._insert_resource`, which configures library-level parameters (f.e. `num_retries`), and handles errors correctly. . See any other `create_` method in this class for example.